{{cookiecutter.project_slug}}/server/{{cookiecutter.project_slug}}/common/models.py

```diff
@@ -16,4 +16,4 @@ class Meta:
         abstract = True
 
     def __str__(self):
-        return "ah yes"
+        return "__str__ not defined for this model"
```

Comment:
gentle nudge for devs to set this for their new models

Reply:
Thank you